lib/rspec/rails/view_rendering.rb

```diff
@@ -44,6 +44,13 @@ def render_views?
       #
       # @private
       class EmptyTemplateResolver < ::ActionView::FileSystemResolver
+
+        def initialize(path, pattern=nil)
+          unless path.is_a?(::ActionView::Resolver)
+            super
+          end
+        end
+
```

Comment:
Well we need to find a way to pass the details of the resolver down into that implementation without loosing this subclass, alternatively we could monkey patch the instance of the resolver with the required methods but I'd rather not do that, that seems dangerous